packages/lib-classifier/src/store/FeedbackStore.spec.js

```diff
@@ -0,0 +1,18 @@
+import FeedbackStore from './FeedbackStore'
+// import { ProjectFactory, SubjectFactory, WorkflowFactory, SingleChoiceTaskFactory } from '../../test/factories'
```

Comment:
Let's trash this if it's not needed

Reply:
If we're testing feedback, then we'll need the subject factory, and also a mock classification? Maybe a workflow too.